Runtime/Client/LootLockerBaseServerAPI.cs

```diff
@@ -170,23 +142,75 @@ IEnumerator coroutine()
+                                        errorData.code = "payment_required";
+                                        break;
+                                    case 403:
+                                        errorData.message = "Forbidden -- You do not have access to this resource.";
+                                        errorData.code = "forbidden";
+                                        break;
+                                    case 404:
+                                        errorData.message = "Not Found -- The requested resource could not be found.";
+                                        errorData.code = "not_found";
+                                        break;
+                                    case 405:
+                                        errorData.message = "Method Not Allowed -- The selected http method is invalid for this resource.";
+                                        errorData.code = "method_not_allowed";
+                                        break;
+                                    case 406:
+                                        errorData.message = "Not Acceptable -- Purchasing is disabled.";
+                                        errorData.code = "not_acceptable";
+                                        break;
+                                    case 409:
+                                        errorData.message = "Conflict -- Your state is most likely not aligned with the servers.";
+                                        errorData.code = "conflict";
+                                        break;
+                                    case 429:
+                                        errorData.message = "Too Many Requests -- You're being limited for sending too many requests too quickly.";
+                                        errorData.code = "too_many_requests";
+                                        break;
+                                    case 500:
+                                        errorData.message = "Internal Server Error -- We had a problem with our server. Try again later.";
+                                        errorData.code = "internal_server_error";
+                                        break;
+                                    case 503:
+                                        errorData.message = "Service Unavailable -- We're either offline for maintenance, or an error that should be solvable by calling again later was triggered.";
+                                        errorData.code = "service_unavailable";
+                                        break;
+                                    default:
+                                        errorData.message = "Unknown error.";
+                                        break;
+                                }
+                            }
+                            errorData.message += " " + ObfuscateJsonStringForLogging(webRequest.downloadHandler.text);
```

Comment:
Not sure this is right. In the case of the informative error messages we don't want the whole body after the message. And will there ever be info needing obfuscation in the returned message.